docs/content/docs/data-structures/counter.md

```diff
@@ -1,5 +1,208 @@
+
+So in this case, both users would see the value 43, rather than 44.
+
+This is a problem!
+If this shared counter value was being used to track store inventory, for example, that inventory would now be incorrect!
+
+The solution to this problem is a specialized DDS that tracks changes to the shared value as _increments_ and _decrements_, which can be summed together in any order and still reach eventual consistency.
+
+{{% callout tip %}}
+
+You may be asking yourself: but wouldn't the counter incrementing by 2 after the user pressed the increment button just once confuse the user?
+How do I make clear to the user that the additional increment was the result of a collaborator's actions?
+
+Check out [User presence and audience]({{< relref "audience.md" >}}) for an overview of Fluid's APIs for working with user presence!
+
+{{% /callout %}}
+
+## Creation
+
+The `FluidContainer` provides a container schema for defining which DDSes you would like to load from it.
+It provides two separate fields for establishing an initial roster of objects and dynamically creating new ones.
+
+- For general guidance on using the `ContainerSchema`, see [Data modeling]({{< relref "data-modeling.md" >}}).
+- For guidance on how to create and load a Fluid container using a service-specific client, see [Containers - Creating and loading]({{< relref "containers.md#creating--loading" >}}).
+
+The following example illustrates how you can use the `ContainerSchema` with `SharedCounter`.
+
+There are two key workflows to consider here.
+
+1. Creating a new Fluid Container.
+2. Connecting to an existing Fluid Container.
+
+In either case, you will first define the schema.
+
+```javascript
+const schema = {
+    initialObjects: {
+        sharedCounter: SharedCounter,
+    }
+}
+```
+
+### Creating a new Fluid Container
+
+Here, you will generate a new  `SharedCounter` as part of the initial roster of objects you have available in the container, using the schema defined above.
+
+```javascript
+const { container, services } = await client.createContainer(schema);
+
+const counter = container.initialObjects.sharedCounter;
+```
+
+At this point, you can directly start using the `counter` object within your application.
+Including the `SharedCounter` as part of initial objects ensures that the DDS is available the moment the async call to `createContainer` finishes.
+
+### Connecting to an existing Fluid Container
+
+Similarly, if you are connecting to an existing container, the process stays largely identical with the only difference being that you use `getContainer` instead of `createContainer`.
+
+```javascript
+const { container, services } = await client.getContainer(id, schema);
+
+const counter = container.initialObjects.sharedCounter;
+```
+
```

Comment:
Here, we need to show how the client code discovers whether the container already exists.

Reply:
Hmm. That's going to be very app-specific. Either they have the ID for a container or they don't. Where that information is stored / how the app determines if it is present is going to vary pretty widely. Do you think just noting the presence or absence of a container ID is sufficient?

Reply:
Per your other suggestion, I went ahead and just pointed readers to the creation workflow outlined in the SharedMap article. But we may want to update *that* document's creation section.